cmd/juju/controller/addmodel_test.go

```diff
@@ -196,7 +196,7 @@ func (s *AddModelSuite) TestInit(c *gc.C) {
 }
 
 func (s *AddModelSuite) TestAddExistingName(c *gc.C) {
-	s.SetFeatureFlags(feature.Generations)
+	s.SetFeatureFlags(feature.Branches)
```

Comment:
I think it would be good to leave one of these as `Generations` with a comment that it tests the either-or condition.

Reply:
good idea, I will add one additional test